src/components/Table/common/gridTable.styles.ts

```diff
@@ -44,6 +45,7 @@ export const GridTable = styled(MTable, {
     background-color: ${smokeLightest};
 
     td {
+      ${textBodySmall500};
```

Comment:
Styled updated as per [mock](https://www.figma.com/design/g60pHZLz78wfl0PvbksMb5/HCA-Tracker?node-id=1019-4782&t=SQWl256TF2lcHXMT-4) for grouped column.